packages/components/Input/src/Input.mobile.tsx

```diff
@@ -0,0 +1,107 @@
+/** @jsxRuntime classic */
```

Comment:
If Input is not planned to Desktop platform we can have only one file as Input.tsx and that should be fine.

Reply:
Chip was in a similar situation, it was recommended there to move it out to a mobile file, following same precedence here.